### PR TITLE
Build with pip

### LINKED ISSUE #2: auto_paridecl.pxd is missing
I get the following error when trying to build Sage 8.1 from conda at https://github.com/conda-forge/sagelib-feedstock/pull/16

```
Error compiling Cython file:
------------------------------------------------------------
...
    void   pari_err_ROOTS0(const char *f)

# Auto-generated declarations. There are taken from the PARI version
# on the system, so they more up-to-date than the above. In case of
# conflicting declarations, auto_paridecl should have priority.
from .auto_paridecl cimport *
^
------------------------------------------------------------

/home/jule/proj/conda/miniconda3/envs/conda-build-sagelib/conda-bld/sagelib_1521489491380/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol/lib/python2.7/site-packages/cypari2/paridecl.pxd:5174:0: 'cypari2/auto_paridecl.pxd' not found
```

Are we missing this auto-generated file?

### PR DESCRIPTION
**Pull request by saraedum**
fixes #2

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,12 +11,12 @@ source:
...
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win]
   script:
     # https://github.com/conda-forge/toolchain-feedstock/pull/11
     - export CPPFLAGS="-I${PREFIX}/include ${CPPFLAGS}"
-    - python setup.py install --single-version-externally-managed --record record.txt
+    - pip install .
```
Comment (isuruf) [inline recipe/meta.yaml]: This should be https://conda-forge.org/docs/meta.html#use-pip

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,12 +11,12 @@ source:
   sha256: b61e30a697250adf6369c21f148f7552aae6abfbb673171c2b97ec3c8fb74676
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win]
   script:
     # https://github.com/conda-forge/toolchain-feedstock/pull/11
     - export CPPFLAGS="-I${PREFIX}/include ${CPPFLAGS}"
```
Comment (isuruf) [inline recipe/meta.yaml]: This has been fixed and can be removed

**Comment by saraedum** [comment]
Thanks for the quick feedback :)

**Comment by saraedum** [comment]
Seems like this worked. Feel free to merge this anytime.

**Pull request merged**